### PR TITLE
Follow cffdump and ignore missing memory for load_state constants

### PR DESCRIPTION
**Pull request by shanminchao**
I'm not sure why this is the case. TODO: Investigate in Turnip driver how this works.

But chromebook capture definitely has captures where this memory block was never captured. And cffdump ignores it,
so we should too.

### CODE REVIEW

File: dive_core/command_hierarchy.cpp [hunk 1]
```diff
@@ -2547,7 +2547,11 @@ void CommandHierarchyCreator::AddConstantsToPacketNode(const IMemoryManager &mem
             {
                 T        value;
                 uint64_t addr = ext_src_addr + ((i + j) * sizeof(T));
-                DIVE_VERIFY(mem_manager.CopyMemory(&value, submit_index, addr, sizeof(T)));
+
+                // For some reason, some captures refer to memory not backed by memory blocks
+                // Let's not treat it as an error, since cffdump handles this gracefully as well
+                if (!mem_manager.CopyMemory(&value, submit_index, addr, sizeof(T)))
```
Comment (wangra-google) [inline dive_core/command_hierarchy.cpp:2553]: should we use `DIVE_LOG` (or maybe other func) to print warning messages in this case?

**Comment by wangra-google** [comment]
> I'm not sure why this is the case. TODO: Investigate in Turnip driver how this works.
> 
> But chromebook capture definitely has captures where this memory block was never captured. And cffdump ignores it, so we should too.

probably because of this: 
image

File: dive_core/command_hierarchy.cpp (hunk 1 above)
Comment (shanminchao) [reply dive_core/command_hierarchy.cpp:2553]: Added log as suggested

**Comment by shanminchao** [comment]
> > I'm not sure why this is the case. TODO: Investigate in Turnip driver how this works.
> > But chromebook capture definitely has captures where this memory block was never captured. And cffdump ignores it, so we should too.
> 
> probably because of this (i think the freedreno shim layer captures everything by default): image

Thanks for explanation!

**Review by wangra-google — APPROVED** [review]

**Pull request merged**